lib/accumulate_distribute/util/gen_order_amounts.js

```diff
@@ -26,7 +26,7 @@ const generateOrderAmounts = (args = {}) => {
 
     while (Math.abs(amount - totalAmount) > DUST) {
       const m = Math.random() > 0.5 ? 1 : -1
-      const orderAmount = Math.min(sliceAmount, sliceAmount * (1 + (Math.random() * amountDistortion * m)))
```

Comment:
One year ago `Math.min` was added in the commit https://github.com/bitfinexcom/bfx-hf-algo/commit/45663868971291b5f75ecb6cd0c7bd40aa4980d0#diff-56ca5cc772b02af17459607b243e9a3a9998659ba3f174b1e1daf03bd38d1a88R17

I assume it fixes the sum of all slices that could be a different value from the specified amount (

For example, without `Math.min` I received (only Sell):

![image](https://user-images.githubusercontent.com/70510980/115214457-33297800-a10b-11eb-8890-39a9dad59218.png)

The sum isn't equal to 60, but it should. @avsek477 @robertkowalski right?

Reply:
yes .. it should ... let me check again.. i tried with many example at last it would calculate whatever was remaining

Reply:
One more example:
![image](https://user-images.githubusercontent.com/70510980/115218972-bf3d9e80-a10f-11eb-9c9d-50dc0e12e5ef.png)

Reply:
after chat discussion: minimum order size amount will be solved in a separate task